Packs/PrismaAccess/Integrations/PrismaAccess/PrismaAccess.py

```diff
@@ -0,0 +1,470 @@
+            XPATH_SECURITY_RULES = "/config/shared/"
+            DEVICE_GROUP = device_group_shared
+        else:
+            XPATH_SECURITY_RULES = "/config/devices/entry/device-group/entry[@name=\'" + DEVICE_GROUP + "\']/"
+    else:
+        XPATH_SECURITY_RULES = "/config/devices/entry/vsys/entry[@name=\'" + VSYS + "\']/rulebase/security/rules/entry"
+
+    # setting objects xpath relevant to FW or panorama management
+    if DEVICE_GROUP:
+        device_group_shared = DEVICE_GROUP.lower()
+        if DEVICE_GROUP == 'shared':
+            XPATH_OBJECTS = "/config/shared/"
+            DEVICE_GROUP = device_group_shared
+        else:
+            XPATH_OBJECTS = "/config/devices/entry/device-group/entry[@name=\'" + DEVICE_GROUP + "\']/"
+    else:
+        XPATH_OBJECTS = "/config/devices/entry/vsys/entry[@name=\'" + VSYS + "\']/"
+
+
+PAN_OS_ERROR_DICT = {
+    '1': 'Unknown command - The specific config or operational command is not recognized.',
+    '2': 'Internal errors - Check with technical support when seeing these errors.',
+    '3': 'Internal errors - Check with technical support when seeing these errors.',
+    '4': 'Internal errors - Check with technical support when seeing these errors.',
+    '5': 'Internal errors - Check with technical support when seeing these errors.',
+    '6': 'Bad Xpath -The xpath specified in one or more attributes of the command is invalid.'
+         'Check the API browser for proper xpath values.',
+    '7': 'Object not present - Object specified by the xpath is not present. For example,'
+         'entry[@name=value] where no object with name value is present.',
+    '8': 'Object not unique - For commands that operate on a single object, the specified object is not unique.',
+    '10': 'Reference count not zero - Object cannot be deleted as there are other objects that refer to it.'
+          'For example, address object still in use in policy.',
+    '11': 'Internal error - Check with technical support when seeing these errors.',
+    '12': 'Invalid object - Xpath or element values provided are not complete.',
+    '14': 'Operation not possible - Operation is allowed but not possible in this case.'
+          'For example, moving a rule up one position when it is already at the top.',
+    '15': 'Operation denied - Operation is allowed. For example, Admin not allowed to delete own account,'
+          'Running a command that is not allowed on a passive device.',
+    '16': 'Unauthorized -The API role does not have access rights to run this query.',
+    '17': 'Invalid command -Invalid command or parameters.',
+    '18': 'Malformed command - The XML is malformed.',
+    # 19,20: success
+    '21': 'Internal error - Check with technical support when seeing these errors.',
+    '22': 'Session timed out - The session for this query timed out.'
+}
+
+
+class PAN_OS_Not_Found(Exception):
+    """ PAN-OS Error. """
+
+    def __init__(self, *args, **kwargs):  # real signature unknown
+        pass
```

Comment:
Just do
```python
class PAN_OS_Not_Found(Exception):
    pass

```
This way you will not override the constructor of Exception class - and the lint will not error
Should fix

![image](https://user-images.githubusercontent.com/7270217/80303734-b47d5180-87ba-11ea-8823-9e5801f68918.png)